reference/7.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

```diff
@@ -982,15 +982,17 @@ using flag notation or combining multiple `WebSslProtocol` options with `-bor`,
 multiple protocols is not supported on all platforms.
 
 > [!NOTE]
-> On non-Windows platforms it may not be possible to supply `'Tls, Tls12'` as an option.
+> On non-Windows platforms it may not be possible to supply `'Tls, Tls12'` as an option. Support for
+> `'Tls13'` is not available on all Operating Systems and will need to be verified on a per
```

Comment:
I know you are just keeping this format from the previous state, but is there a reason for the codefence and ticks together?

Reply:
Actually no. I think code fence is going to benthe best option here.

Reply:
fixed